src/main/java/dev/vasyl/proj/service/BookServiceImpl.java

```diff
@@ -11,13 +15,25 @@
 public class BookServiceImpl implements BookService {
     private final BookRepository bookRepository;
 
+    private final BookMapper bookMapper;
+
+    @Override
+    public BookDto save(CreateBookRequestDto bookDto) {
+        Book book = bookRepository.save(bookMapper.toModel(bookDto));
+        return bookMapper.toDto(book);
+    }
+
     @Override
-    public Book save(Book book) {
-        return bookRepository.save(book);
+    public List<BookDto> findAll() {
+        return bookRepository.findAll().stream()
+                .map(bookMapper::toDto)
+                .toList();
```

Comment:
FYI if you want you can declare mapping List to List in mapstruct interface instead of stream map